runtime/syntax/testdir/input/tex_02.tex

```diff
@@ -0,0 +1,11 @@
+\documentclass{article}
+\newcommand{\foo}{}
+\renewcommand*{\foo}[1]{#10}
+\providecommand{\foo}[1][default]{#11}
```

Comment:
TODO: `#1` is not highlighted as texDefParm, and `1` is not perceived as texCmdBody.